index.bs

```diff
@@ -2339,7 +2339,7 @@ optionally evidence of [=user consent=] to a specific transaction.
 
 <xmp class="idl">
     dictionary PublicKeyCredentialParameters {
-        required PublicKeyCredentialType      type;
+        required DOMString                    type;
```

Comment:
The corresponding one in `credTypesAndPubKeyAlgs` in [`authenticatorMakeCredential`](https://pr-preview.s3.amazonaws.com/jcjones/webauthn/pull/1392.html#sctn-op-make-cred) can remain a strict enum, right?

Reply:
I think that's fine. By the time we're running the Authenticator-level algorithm, the User Agent has already ignored unknown values.

Reply:
Yep, sounds good!